services/account/libs/common/src/types/dtos/accounts.request.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { TransactionType } from '#lib/types/enums';
+import { ExtrinsicPayload, Signature } from '@polkadot/types/interfaces';
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty } from 'class-validator';
+import { SignerPayloadJSON } from '@polkadot/types/types';
+
+export class RetireMsaRequestDto {
+  @ApiProperty({ type: Object })
+  @IsNotEmpty()
+  unsignedPayload: SignerPayloadJSON;
+
+  @ApiProperty({ type: String })
+  @IsNotEmpty()
+  encodedPayload: ExtrinsicPayload;
```

Comment:
In general, I don't think we should use types from the `@polkadot/*` libraries directly in our REST APIs. The reason being twofold:
1. The types are subject to change outside of our control, thus changing our API potentially unintentionally
2. The types are outside our control; we don't have the ability to add `@ApiProperty()` decorators to them & therefore our generated OpenAPI spec is opaque when it comes to these fields
3. These types are SCALE-encoded types; although there is some work going on under the hood to convert to standard JSON non-SCALE representation when we respond, in general it's not a good practice, as we're trying to isolate the client from having exposure to SCALE concerns.

I'd suggest creating our own types for any request/response objects, and if we want to marshal to/from the `@polkadot/*` types, we should create corresponding `to...` and `from...` utility methods.